src/skmultiflow/demos/_test_mtr.py

```diff
@@ -1,20 +1,18 @@
-from skmultiflow.core.pipeline import Pipeline
 from skmultiflow.data import RegressionGenerator
 from skmultiflow.evaluation.evaluate_prequential import EvaluatePrequential
-from skmultiflow.trees.multi_target_regression_hoeffding_tree import \
-    MultiTargetRegressionHoeffdingTree
+from skmultiflow.trees.isoup_tree import iSOUPTreeRegressor
```

Comment:
Would not be better to import `iSOUPTreeRegressor` directly from the `trees` submodule?

Reply:
Agree. Updated in fdc3750b4e53c4c24cfd9b1d5127107543969262